scripts/optimize_images/static-images.ts

```diff
@@ -1,28 +1,100 @@
-const staticImages = [
+import { chain, concat } from 'lodash';
+import { teams } from '../../src/cms-content/team';
+import aboutContent from '../../src/cms-content/about';
+import { allCaseStudies } from '../../src/cms-content/learn';
+import {
+  ImageInfo,
+  isRasterImage,
+  joinPublicFolder,
+  cmsBackupPath,
+} from './utils';
+
+const staticImages: ImageInfo[] = [
   {
-    path: 'src/assets/images/ghss.png',
+    inputPath: 'src/assets/images/ghss.png',
+    backupPath: 'src/assets/images/ghss-original.png',
```

Comment:
`backupPath` makes it sound like it's just a backup copy or something.  Perhaps `originalPath` ?